RubeesEat/wwwroot/css/site.css

```diff
@@ -0,0 +1,128 @@
+/*source: https://www.joshwcomeau.com/css/custom-css-reset/*/
+
+:root {
+    --primary-color: #fbbf00;
+    --secondary-color: #2d332f;
+}
+
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+body {
+    line-height: 1.5;
+    -webkit-font-smoothing: antialiased;
+    margin: 0;
+    font-family: "Metric Regular", Arial, Tahoma, sans-serif;
+    text-align: center;
+    background-color: var(--secondary-color);
+    color: white;
+}
+
+img, picture, video, canvas, svg {
+    display: block;
+    max-width: 100%;
+}
+
+input, button, textarea, select {
+    font: inherit;
+}
+
+p, h1, h2, h3, h4, h5, h6 {
+    overflow-wrap: break-word;
+    text-align: center;
+}
+
+nav {
+    background-color: var(--primary-color);
+    padding: 1em 2em;
+    position: sticky;
+    top: 0;
+    z-index: 1000;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+nav > ul {
```

Comment:
On mobile, the elements are not centered correctly. On desktop, we want the elements next to the logo on the left side with the right side being empty.